lib/components/user/saved-trip-screen.js

```diff
@@ -102,12 +103,35 @@ class SavedTripScreen extends Component {
 
   _handleSaveNewTrip = async () => {
     await this._updateMonitoredTrip()
-    this._goToTripPlanner()
+
+    // For new trips, if a trip with the same ui_activeSearch value in queryParams
+    // appears in the monitored trips redux state,
+    // then we know that save was successful and we can proceed.
+    const { activeSearchId } = this.props
+
+    const savedTrip = this.props.monitoredTrips.find(trip => {
+      const tripQueryParams = qs.parse(trip.queryParams.split('?')[1])
+      return tripQueryParams.ui_activeSearch === activeSearchId
+    })
+
+    if (savedTrip) {
+      this._goToTripPlanner()
+    }
```

Comment:
I think I mentioned in a recent PR that we probably shouldn't go to the trip planner after saving the trip, but rather the list of saved trips. I can't recall whether that was already merged/implemented.

Reply:
This was handled in #239 (see this file: https://github.com/opentripplanner/otp-react-redux/blob/cf194ec2d2117caf7d60b47fadc7724c606d5800/lib/components/user/saved-trip-screen.js#L99)